models/marts/dag/fct_too_many_joins.sql

```diff
@@ -0,0 +1,26 @@
+with all_dag_relationships as (
+    select
+        *
+    from {{ ref('int_all_dag_relationships') }}
+    where not child_is_excluded
```

Comment:
```suggestion
    where not child_is_excluded
    and child_resource_type = 'model'
```
I think we only want to include models here?